### DISCUSSION TITLE
Unique STAR run when running multiple chunks

### DISCUSSION THREAD

**Discussion by MartinezRuiz-Carlos** [ConesaLab/SQANTI3#293] [Ideas] [answered]
I find that the chunk parallelisation for the QC step with -n works very well, however, it becomes unusable when running the alignment with STAR. Giving multiple chunks will try and run STAR (index and alignment) as many times as chunks provided. This seems unnecessary and results in very large amounts of memory being requested. Would it be possible to have a single STAR run regardless of the number of chunks requested in the -n argument?
Many thanks!

**Comment by carolinamonzo** [comment]
Moved to issues since it looks like it should be revised by developers and update SQANTI3 with changes if necessary.

**Reply by carolinamonzo** [reply, marked as answer]
This problem was solved in f77e878
